README.Rmd

```diff
@@ -9,30 +9,91 @@ output:
 
 ```{r, echo = FALSE}
 knitr::opts_chunk$set(
-  collapse = TRUE,
-  comment = "#>",
-  fig.path = "figs/README-"
+    collapse = TRUE,
+    comment = "#>",
+    fig.path = "figs/"
 )
 ```
 
 # MoMA: Modern Multivariate Analysis
 
-[![Travis-CI Build Status](https://api.travis-ci.com/michaelweylandt/MoMA.svg)](https://travis-ci.com/michaelweylandt/MoMA) [![License: GPL v2](https://img.shields.io/badge/License-GPL%20v2-brightgreen.svg)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) [![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/MoMA)](https://cran.r-project.org/package=MoMA) [![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](http://www.repostatus.org/badges/latest/wip.svg)](http://www.repostatus.org/#wip)
+<!-- badges: start -->
+[![Travis-CI Build Status](https://api.travis-ci.com/michaelweylandt/MoMA.svg)](https://travis-ci.com/michaelweylandt/MoMA) 
```

Comment:
I think these links need to refer to `DataSlingers` instead of `michaelweylandt` now.

Reply:
Fixed. See commit "Update badges".